### ISSUE TITLE
No Vanilla Mobs

### ISSUE THREAD

**Issue by Burgersim** [Thutmose/Pokecube#323]
#### Feature being requested

No more Vanilla Mobs - being able to get all Vanilla resources from the Pokemobs

#### Details on feature/suggestion

I would really love to have the option to disable vanilla mobs and have only the Pokemobs in my world. I'm not sure if this is already possible, if so then sorry for requestiong it and I hope somebody can tell me how. And of course it would be nice to get all the needed vanilla resources from pokemobs too, like milk from miltanks or other cow-like pokemon, wool from sheep-like pokemon like mareep/flaaffy, rotten flesh (i'm not sure xD), feathers and eggs from bird-like pokemon, and so on and so forth.

**Comment by Burgersim** [comment]
I think I already saw that there is a config option for disabling the vanilla animals and monsters. My only other question would be, are all thhe vanilla mob drops obtainable from pokemobs?

**Comment by Thutmose** [comment]
yes, they should all be obtainable from pokemobs, you can also edit drops via loottables or configs.
